ckanext/csrf_filter/anti_csrf.py

```diff
@@ -90,6 +91,9 @@ def configure(config):
     token_expiry_age = 60 * config.get('ckanext.csrf_filter.token_expiry_minutes', 30)
     token_renewal_age = 60 * config.get('ckanext.csrf_filter.token_renewal_minutes', 10)
 
+    exempt_rules = config.get('ckanext.csrf_filter.exempt_rules', None)
+    if exempt_rules:
```

Comment:
Seems like a good option. This does warrant an actual unit test which would also show examples as something this 'powerful' left without proper unit testing/examples can/will shoot people in the foot for overly lax regex rules etc.

Reply:
A warning has been added to the readme for this config option telling users to make sure they do strict regex statements.

I have added in a new test which shows a use case with the Datatables view plugin. Which was my reasoning for doing exempt rules to begin with as the Datatables stuff happens in JS and not HTML so no token fields get injected to it